site/operational-guide/npc.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Weave NPC
+menu_order: 70
+---
+
+* ["Connection dropped by Weave NPC"](#connection-dropped)
+
+##<a name="connection-dropped"></a>"Connection dropped by Weave NPC"
+
+When using Weave together with [Kubernetes](http://kubernetes.io) and [Weave NPC](https://github.com/weaveworks/weave-npc), Weave's Kubernetes Network Policy Controller, `weave launch --expect-npc` sets up the two below default `iptables` rules, in case none of the NPC rules apply:
```

Comment:
About the "how it works" section, I am happy to try this, but given I have never used Kubernetes and never worked on Weave NPC, I may not be the best person to write this at the moment, and this may take me a while to get there. 
I'll look into it after I'm done with other issues / PRs.